LayoutTests/interaction-region/svg-lumincance-expected.txt

```diff
@@ -0,0 +1,41 @@
+(GraphicsLayer
+  (anchor 0.00 0.00)
+  (bounds 800.00 600.00)
+  (children 1
+    (GraphicsLayer
+      (bounds 800.00 600.00)
+      (contentsOpaque 1)
+      (drawsContent 1)
+      (backgroundColor #FFFFFF)
+      (event region
+        (rect (0,0) width=800 height=600)
+
+      (interaction regions [
```

Comment:
We should invent a ref test mode that actually paints these regions in the result. Diffing/reviewing these paths is next to impossible.

Reply:
excellent idea, making sure it's captured!